API.md

```diff
@@ -2600,6 +2600,29 @@ server.ext('onRequest', function (request, reply) {
 });
 ```
 
+#### `request.generateResponse(source, [options])`
+
+_Always available._
+
+Returns an [`response`](#response-object) which you can pass into the [reply interface](#reply-interface).  For example inside a promise to create a response object to return up to the [reply interface](#reply-interface).
```

Comment:
Returns **a**...

Move the "For example" to after the arguments before the example and end the first sentence with "where:" like we do elsewhere int he document. Also no empty line before the arguments.